actiontext/lib/action_text/attribute.rb

```diff
@@ -56,12 +59,26 @@ def #{name}
           def #{name}?
             rich_text_#{name}.present?
           end
-
-          def #{name}=(body)
-            self.#{name}.body = body
-          end
         CODE
 
+        if store_if_blank
+          class_eval <<-CODE, __FILE__, __LINE__ + 1
+            def #{name}=(body)
+              self.#{name}.body = body
+            end
+          CODE
+        else
+          class_eval <<-CODE, __FILE__, __LINE__ + 1
+            def #{name}=(body)
+              if body.present?
+                self.#{name}.body = body
+              else
+                self.#{name}.mark_for_destruction if #{name}?
```

Comment:
Can we still add 
```ruby
# ...
else
  self.#{name}.body = body # add this as well
  self.#{name}.mark_for_destruction if #{name}?
end
```

because it's thanks to this line that I can add `validates :content, presence: true` and make the presence validation work in my AR model.

With this change I would need to add a custom validator `errors.add(....) if self.content.marked_for_destruction?` ( I think)

Unless you know of a documented way to enforce presence of a `has_rich_text`

Reply:
Thanks @chaadow made this PR: https://github.com/rails/rails/pull/52709